v4/newrelic/application.go

```diff
@@ -28,8 +30,11 @@ func (app *Application) StartTransaction(name string) *Transaction {
 	return &Transaction{
 		rootSpan: s,
 		thread: &thread{
-			currentSpan: s,
+			currentSpan:  s,
+			isSingleSpan: true,
```

Comment:
Nitpick. If instead we use a bool that starts as `false` and moves to `true` when a span is started, we can rely on the default value of the bool. This might be helpful if we were to accidentally not include this field in the creation of a new `*thread`.

Reply:
Done. It is now `isMultiSpan`.